src/resources/OrganizationAccess/AccessInterfaces.ts

```diff
@@ -22,9 +22,32 @@ export interface AccessParams {
 }
 
 export interface AccessModel {
-    accessLevel: AccessLevel[];
-    callerPartOf: boolean;
+    /*
+     * The access level for this privilege.
```

Comment:
The language around privileges and access models can be quite confusing. Since we're in the context of an access model here, I would advise against mentioning the word "privilege" in any of the properties of the AccessModel interface, since privileges are an entirely different (but related) beast.

```suggestion
     * The access level granted by the access model.
```